application/src/org/commcare/entity/RecentFormEntity.java

```diff
@@ -56,11 +56,11 @@ public boolean match (String key) {
         return true;
     }
 
-    public int[] getStyleHints (boolean header) {
+    public String[] getStyleHints (boolean header) {
         if(header) {
-            return new int[] {-1, -1} ;
+            return new String[] {null, null} ;
         } else {
-            return new int[] {-1, 36 };
+            return new String[] {null, "36%" };
```

Comment:
@ctsims Do you know offhand what this code is for? J2ME case list header row has a 36% column? I'm looking at https://confluence.dimagi.com/download/attachments/10223880/case%20select.png?version=1&modificationDate=1342853926863&api=v2 but not sure what the 36 is for.